src/main/java/uk/gov/hmcts/darts/arm/client/ArmRpoClient.java

```diff
@@ -1,12 +1,22 @@
 package uk.gov.hmcts.darts.arm.client;
 
 import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestHeader;
+import uk.gov.hmcts.darts.arm.client.model.rpo.RecordManagementMatterResponse;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
 
 @FeignClient(
     name = "arm-rpo-client",
     url = "${darts.storage.arm-api.url}"
 )
 public interface ArmRpoClient {
 
-
+    @PostMapping(value = "${darts.storage.arm-api.rpo-url.get-record-management-matter-path}",
```

Comment:
Is `rpo-url` incorrect, should it be this?

```suggestion
    @PostMapping(value = "${darts.storage.arm-api.api-url.get-record-management-matter-path}",
```

Reply:
Ignore, I see the new value in application.yaml.